libs/sdk-ui-tests/scenarios/charts/columnChart/base.tsx

```diff
@@ -58,6 +60,20 @@ export const ColumnChartViewByDateAndPop = {
     viewBy: [ReferenceLdm.ClosedYear],
 };
 
+const modifiedClosedYear: IAttribute = modifyAttribute(ReferenceLdm.ClosedYear, (m) =>
```

Comment:
Since this is used in multiple tests we should probably consider specifying it in ReferenceLdmExt in the reference-workspace package so that we can use it everywhere from one place.

Reply:
done [commit](https://github.com/gooddata/gooddata-ui-sdk/pull/1444/commits/b837e03939320efd8b7a17dfb16ccfe3a441611d)